### ISSUE TITLE
Create actual design & UI

### ISSUE THREAD

**Issue by wslyvh** [wslyvh/tokenlog#10]
Few ideas on UI/UX design & branding.

I’d like to see a more coherent design through the entire product (a logo, brand, and website/App design).

- Primary audience: Github users and (active) token-holders who’re engaged in their respective projects and communities. This means they’ll have a basic understanding of:
    - Git and Github, product backlog management, product and community roadmap, web3, wallet & token/contract interactions
- The design should be minimal, clean, and easy to use, optimized for a) the described target audience and b) the core functionalities of tokenlog:
    - have a community prioritized product backlog - that provides an overview of the top-items to work on next
    - allow members to quickly browse, find (and create - but we can use Github for that) backlog items 
    - allow members to vote and help prioritize the product backlog 
- Design should be based on [Github’s Primer design system](https://primer.style/). It should feel like an extension/add-on of Github.
- The Tokenlog App/UI should be easy to use and provide a clear overview and prioritized backlog. Not another administrative tool to manage.
- Github will remain the main source of truth. Tokenlog should be integrated as closely as possible. Some ideas here are to use Github bots/Apps (e.g. #13) so you can manage from Github and not require the Tokenlog App/UI at all. Also see the previous point.

Looking to design it in an iterative approach, aligned with development and on real-world usage and feedback from projects 

Some inspiration for managing roadmaps & customer feedback:
* https://snapshot.page/#/piedao
* https://roadmap.zerion.io/ (conflux)
* https://safe.cnflx.io/boards/gnosis-safe-multisig (upvoty)
* https://feedback.azure.com/forums/34192--general-feedback (uservoice)
* https://feedback.mongodb.com/forums/924145-atlas

**Comment by johnnthough** [comment]
Hello i am a UI designer and i believe i can be of help to you

**Comment by wslyvh** [comment]
Hi, that's great to hear! Besides the mentioned links above, I haven't thought about it too much yet.. 
A lot is depending on some of the user interviews that I'm currently doing 

Do you have any ideas, or previous work that you can share, to see how we can approach this? 

cc @bennisan86 - who might join in here as well. Hope we can collaborate in some way!

**Comment by johnnthough** [comment]
looking at the links i can say you need a UI design for dropping customer feedback yes? i have not designed something like that before but here is a link for an FAQ page i designed i think its the closest thing i have done yet 
https://perxels.com/faq

**Comment by wslyvh** [comment]
Yes, a design for the App and to gather customer feedback and manage a backlog is one.
In the long term, I'd like to see a complete design for Tokenlog (logo, landing page, design system, etc.) for the overall experience.

Are you part of that design school? Would definitely be cool to collaborate!

**Comment by johnnthough** [comment]
yes, i am alumni now though, is there a medium we can communicate more privately my email is davecole006@gmail.com

**Comment by wslyvh** [comment]
I'm trying to keep things as much as possible here on Github. As I want to set-up Sourcered here soon, to reward contributions. See #14 

But will connect by mail. 
PS. You might want to delete your email here before spambots pick it up. I have your emaill! :)

**Comment by wslyvh** [comment]
Planning to use Github's Primer Design System to align closely with the Github look & feel
https://primer.style/

**Comment by arthurp1** [comment]
Hi Wesley, I'm willing to help out with designing a logo, landing page, and UI layout. My email is arthur.poot1@gmail.com (also a Dutchie from Amsterdam). Looking forward to connect!

**Comment by mnvvr** [comment]
Hello, I would like to contribute by helping designing this. I can do full branding and responsive user interface design based on Github's Primer. Let me know if you still need a hand.

I also opened [this ](https://github.com/wslyvh/tokenlog/issues/45) before noticing this issue.

**Comment by wslyvh** [comment]
Hi, thanks again for reaching out 🙏

We're close to launching a new release of Tokenlog, using Github's Primer.
This already helped us a lot, to make use of their design system and existing components, instead of trying to re-invent, and design our own. 
See preview here at: https://tokenlog.vercel.app/

I'm kind of ok with the overall page structure/layout just resembling Github.
Other than that, plenty of opportunities to improve and we could definitely use someone with your skills :) 
The most value could probably come from designing the homepage. 

Happy to chat more if you want to discuss!

**Comment by mnvvr** [comment]
Hi, 

I reviewed the new interface, better than the current one indeed. Except the homepage; it needs more love. So let me start working on homepage for now and after that I can work on improving repository pages. I can start as soon as you approve.

**Comment by wslyvh** [comment]
Yes please! Go for it.
Here's a link to Github' Primer: https://primer.style/

Let me know if you need any more details, or info. Otherwise, really looking forward to what you can come up with :)

**Comment by mnvvr** [comment]
Hello Wesley,

Heartbeat update: I'm still working on it. I think it will be done before August.

**Comment by wslyvh** [comment]
Awesome! Really looking forward to it...
If there's anything you'd like to review or discuss in the meantime, let me know. Happy to jump on a call :)

**Comment by mnvvr** [comment]
Finally, it has arrived: https://www.figma.com/file/c80pn3tXfVQYrPcgy3eWP5/TOKENLOG?node-id=45%3A1339

Sorry for taking so long, not because I did not care but I actually had a lot of trouble the past few weeks. Let me know what you think, I can revise according to your feedback if necessary.

**Comment by wslyvh** [comment]
Hi @mnvvr, this is great! I really like that you added themes (dimmed mode ftw)

I've been pushing the next release myself now for a while as well. So no worries for taking the time.. We'll get there, eventually. 
Will definitely keep you posted, once I start implementing the design

**Comment by mnvvr** [comment]
Happy to see you like it! I would be delighted to work on the rest of the application too. Would you like me to?

**Comment by mnvvr** [comment]
@wslyvh Ping

**Comment by wslyvh** [comment]
Hi, thanks for following up. And apologies for not replying sooner.

I've been busy with some other projects and haven't had much time to work on Tokenlog.
It seems like that will be the case for a little while longer.
I'd really love to get the rest of the application and design done, but I don't want you to spend more time if it's not used/implemented yet.. If you want to go for it, feel free. But bear in mind that it might take a while to get implemented.
Otherwise, we can maybe reconnect once development on Tokenlog picks up again?

**Comment by mnvvr** [comment]
Hi Wesley, OK then, let’s reconnect once development on Tokenlog picks up again.
